infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -441,11 +433,6 @@ public void handleCommitLayoutRequest(@Nonnull RequestMsg req, @Nonnull ChannelH
             return;
         }
 
-        if (!isBootstrapped(req)) {
-            r.sendNoBootstrapError(req.getHeader(), ctx);
-            return;
-        }
-
```

Comment:
I think this is a bug. See how this check happens after `if (payload.getForced())` , but you are reversing the order. So if it's not bootstrapped, you cant force a bootstrap.